fenix/app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkView.kt

```diff
@@ -137,6 +138,8 @@ class BookmarkView(
     private val bookmarkAdapter = BookmarkAdapter(binding.bookmarksEmptyView, interactor)
 
     init {
+        bookmarkAdapter.setStateRestorationPolicy(RecyclerView.Adapter.StateRestorationPolicy.PREVENT_WHEN_EMPTY)
```

Comment:
It looks like we can just use a property access syntax for this:
`bookmarkAdapter.stateRestorationPolicy = RecyclerView.Adapter.StateRestorationPolicy.PREVENT_WHEN_EMPTY`